client/my-sites/backup/rewind-flow/rewind-config-editor.tsx

```diff
@@ -26,23 +26,15 @@ const BackupRewindConfigEditor: FunctionComponent< Props > = ( {
 	const checkboxes = [
 		{
 			name: 'themes',
-			label: translate( '{{strong}}WordPress Themes{{/strong}}', {
+			label: translate( '{{strong}}WordPress themes{{/strong}}', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.